### ISSUE TITLE
Support for parameter `styles`

### ISSUE THREAD

**Issue by BierDav** [bkbnio/kompendium#610]
There are already todos in the code for this feature:
- https://github.com/bkbnio/kompendium/blob/8a94f6e498697b7b82f36d1570b60d899092afe6/oas/src/main/kotlin/io/bkbn/kompendium/oas/payload/Header.kt#L24
- https://github.com/bkbnio/kompendium/blob/8a94f6e498697b7b82f36d1570b60d899092afe6/oas/src/main/kotlin/io/bkbn/kompendium/oas/payload/Parameter.kt#L29

What is causing that this is not implemented yet? Are there technical issues? bIt seams it should only be one enum with predefined values.

**Comment by brizzbuzz** [comment]
nah iirc there was some strange wording in the openapi spec that made me not entirely sure what the full expectation was for this field.  

feel free to implement the feature, but as mentioned in the other issue, publishing this lib is indefinitely blocked because the maven people revoked my publishing creds as part of some kind of infrastructure upgrade on their end

**Comment by BierDav** [comment]
Okay, that's alright so If you want I can publish it on my maven repository if you don't get yours fixed. Mine works fine. By the way, what about the GitHub repository doesn't this work either?

**Comment by brizzbuzz** [comment]
not sure what you mean, publishing to github repositories work fine, the maven central team just broke my api keys and have been a pain to get back up and running.  

TBF i havent tried very hard, i no longer use Kotlin day to day so find maintaining this repo to be mostly a chore.  

youre of course welcome to publish a fork of the lib to your own repository :) thats the beauty of open source

**Comment by BierDav** [comment]
Ja, but every fork confuses developers exponentialy. Therefore i wouldn't recommend another fork!
